### ISSUE TITLE
Footprints

### ISSUE THREAD

**Issue by spotlesscoder** [RoboUlbricht/kicad.lib#2]
Can you please add footprints so I can make the PCB layout? 

At least I need one for the DFPLayer Mini

**Comment by gmli** [comment]
Yes please. Footprint seems to be unavailable for this module on Internet.
